frontend/src/components/smart/PvPArenaInfo.vue

```diff
@@ -1,53 +1,101 @@
 <template>
   <div>
-    <h1 class="title">ARENA INFORMATION</h1>
+    <h1 class="title">
+      {{$t('pvp.arenaInformationCaps')}}
+    </h1>
     <div class="tokenCard">
       <img src="../../assets/skillToken.png" alt="skill token" />
       <div class="tokenCardInfo">
-        <span class="text">PVP Rewards Pool ($SKILL)</span>
+        <span class="text">
+          {{$t('pvp.rewardsPool')}}($SKILL)
+        </span>
         <span class="number">{{ formatedTierRewardsPool }}</span>
       </div>
     </div>
     <ul class="topPlayersList">
       <li class="header">
-        <span>Top Players</span><span>Rank</span>
+        <span>
+          {{$t('pvp.topPlayers')}}
+        </span>
+        <span>
+          {{$t('pvp.rank')}}
+        </span>
       </li>
       <li>
-        <span>Rank 1: {{ tierTopRankers[0] && tierTopRankers[0].name || 'N/A' }}</span>
-        <span>{{ tierTopRankers[0] && tierTopRankers[0].rank || 'N/A' }}</span>
+        <span>
+          {{$t('pvp.rank')}} 1: {{ tierTopRankers[0] && tierTopRankers[0].name || 'N/A' }}
+        </span>
+        <span>
+          {{ tierTopRankers[0] && tierTopRankers[0].rank || 'N/A' }}
+        </span>
       </li>
       <li>
-        <span>Rank 2: {{ tierTopRankers[1] && tierTopRankers[1].name || 'N/A' }}</span>
-        <span>{{ tierTopRankers[1] && tierTopRankers[1].rank || 'N/A'}}</span>
+        <span>
+          {{$t('pvp.rank')}} 2: {{ tierTopRankers[1] && tierTopRankers[1].name || 'N/A' }}
```

Comment:
'N/A' can be a translation key as well, but don't sweat it, you can totally leave this one, I won't hunt you :D